packages/earth-shared/src/components/multiselect/AsyncSelect.tsx

```diff
@@ -38,7 +38,9 @@ interface AsyncSelectProps {
 const AsyncSelect = (props: AsyncSelectProps) => {
   const { loadFunction, type, selectedGroup, onChange, className, isMulti, ...rest } = props;
   const { t } = useTranslation('admin');
+  const selectRef = useRef();
```

Comment:
```suggestion
  const selectRef: any = useRef();
```
This would allow you to remove the `@ts-ignore` in `hasMoreOptionsAvailable`